### ISSUE TITLE
Add Backpacks As Providers For Botania's Rod of the Shifting Crust

### ISSUE THREAD

**Issue by yrsegal** [gr8pefish/IronBackpacks#38]
Putting a Rod of the Depths/Lands inside won't give you infinite cobble/dirt for your rod.

**Comment by gr8pefish** [comment]
This is not on my end, you need to tell Vazkii to update her API to include this sort of functionality if you want it, I cannot do what you are talking about with her current code. I will also bug her about adding more, so that I can add equipped backpacks to the IBlockProvider list, because that doesn't even have a valid function yet, and my last post about it was deleted without a comment.

**Comment by yrsegal** [comment]
Yes you can, iterate through the items and check for IBlockProviders.

**Comment by gr8pefish** [comment]
Yeah I guess so, just doesn't seem like a very elegant solution. I will do so for now though, and try and think of a better long-term solution.

**Comment by gr8pefish** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add Botania Integration](https://trello.com/c/nuc5UcMd/45-add-botania-integration)
